### PR TITLE
fix nav structure

### PR DESCRIPTION
**Pull request by ryelo**
Jira
https://projects.engineering.redhat.com/browse/RHCLOUD-4310

Nav structure should look like:
https://marvelapp.com/5dcjg6g/screen/63628276

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -356,6 +356,14 @@ rbac:
     title: User Access Management
     paths:
       - /settings/rbac
+    sub_apps:
+      - id: groups
+        title: Groups
+        default: true
+      - id: roles
+        title: Roles
+      - id: Users
```
Comment (karelhala) [inline main.yml]: Lower case id and upper case title.

**Review by karelhala — CHANGES_REQUESTED** [review]

File: main.yml (hunk 1 above)
Comment (ryelo) [reply main.yml]: good catch 😄

**Review by karelhala — APPROVED** [review]

**Review by kruai — APPROVED** [review]

**Comment by ryelo** [comment]
Keeping this on hold. There's a question about whether to use `User access` or `User access management`

**Review by karelhala — APPROVED** [review]

**Review by kruai — APPROVED** [review]

**Comment by kruai** [comment]
LGTM

**Review by kruai — APPROVED** [review]
lol

**Pull request merged**